extensions/capture/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "capture",
+  "title": "Capture",
+  "description": "Quickly capture to your Capture Inbox with Raycast.",
+  "icon": "command-icon.png",
+  "author": "syhchen",
+  "owner": "eleventhlabs",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Capture to Inbox",
```

Comment:
Again just nitpicking but considering how Raycast extensions and commands are named, I think it would make more sense to call it: "Capture record to Inbox" or only "Capture record"